code/transportation-development-services/tds.js

```diff
@@ -506,22 +489,120 @@ function save_Record_ID(recordID) {
           $.ajax(ajaxObject);
         }, timeout);
       } else {
-        console.log("Failed to Capture Record ID");
+        console.log("Failed to Capture Record ID"); // Failure Message in Console
       }
     }
   });
 }
 
-// Listen for record creation (Request Application Submit / First Form)
-// The comment line below also works
-//$(document).on('knack-form-submit.view_731' , function(event, view, record) {
-$(document).on('knack-record-create.view_731', function(event, view, record) {
+// Listen for record creation (TIA Request Application Submit / First Form)
+$(document).on('knack-record-create.view_393', function(event, view, record) {
   const recordId = record.id;
-  save_Record_Id(recordId);
+  console.log('CREATE')
+  saveRecordId(recordId);
 });
 
+/*
```

Comment:
we can delete this, I was checking if the create and submit happened at the same time